cmd/extensions/main.go

```diff
@@ -170,6 +171,21 @@ func main() {
 		health = healthcheck.NewHandler()
 	}
 
+	podReady = true
+	health.AddReadinessCheck("agones-extensions", func() error {
+		if !podReady {
+			return errors.New("Pod is not in Ready state")
```

Comment:
I would make this error clearer and say something like "Pod shutting down, failing readiness checks". The error you have here is tautological.

Reply:
Done!